python/ray/rllib/policy/torch_policy.py

```diff
@@ -53,6 +54,8 @@ def __init__(self, observation_space, action_space, model, loss,
         self.device = (torch.device("cuda")
                        if bool(os.environ.get("CUDA_VISIBLE_DEVICES", None))
                        else torch.device("cpu"))
+        if torch.cuda.device_count() > 1:
+            model = nn.DataParallel(model)
```

Comment:
Nice, any speedups on large models?

I believe one limitation of this approach is that the loss ops are still be run on a single device. However this might not be a big deal if those are lightweight.

Reply:
In versions Im trying out (so [WIP]) -- moving the loss to DataParallel causes issues because I believe apply gradients calls them from the `cpu`.